docs/storefront/graphql/examples/products.mdx

```diff
@@ -598,6 +598,159 @@ You can query product images at different resolutions. The following query retri
   </Tab>
 </Tabs>
 
+## Get product videos
+
+Query the product videos and all configurable media types for products. The API returns images uploaded in the control panel, image URLs added to a product using the control panel, and video URLs linked to a product. 
+
+The following example retrieves video titles and URLs associated with videos attached to a product. 
+
+<Callout type="info">
+Before performing this example, you must ensure to add videos to the product node.
+</Callout>
+
+<Tabs items={['Request', 'Response']}>
+<Tab>
+```graphql filename="Example query: Get product videos for a product" showLineNumbers copy
+  query {
+    site {
+      products (entityId: [112, 113]) {
+        pageInfo {
+            startCursor
+            endCursor
+        }
+        edges {
+              cursor
+              node {
+                  entityId
+                  name
+                  productOptions {
```

Comment:
Adding in product options adds a lot to the example. Is there a reason we have this in here?

Reply:
removed